README.md

```diff
@@ -26,6 +26,15 @@ git push origin 1.2.29
 ```
 Then you need to [create a new release](https://github.com/feedzai/feedzai-openml-java/releases/new) with this tag and the description according [to the previous ones](https://github.com/feedzai/feedzai-openml-java/releases).
 
+### Releasing an Experimental
```

Comment:
I think that with this format, that XP proc can still be ovelooked.
Here are some suggestions:

## Releasing
This section describes the procedure for public releases and experimental releases.
### Public Release
For all releases, as the hotfix branch is ready a..
### Releasing an Experimental ---> Experimental
For experimental releases...

Reply:
Great suggestion!